pquery-run.sh

```diff
@@ -460,74 +501,46 @@ if [[ $PXC -eq 1 ]]; then
   echo "wsrep-debug=1" >> ${BASEDIR}/my.cnf
   echo "innodb_file_per_table" >> ${BASEDIR}/my.cnf
   echo "innodb_autoinc_lock_mode=2" >> ${BASEDIR}/my.cnf
-  if ! check_for_version $MYSQL_VERSION "8.0.0"; then
-    # TODO: research why is this here?
-    # MariaDB: [ERROR] /test/MD300420-mariadb-10.5.3-linux-x86_64-opt/bin/mysqld: unknown variable 'innodb_locks_unsafe_for_binlog=1'
-    # echo "innodb_locks_unsafe_for_binlog=1" >> ${BASEDIR}/my.cnf
-    # echo "wsrep_sst_auth=$SUSER:$SPASS" >> ${BASEDIR}/my.cnf
-    echo "TODO"
-  else
-    echo "log-error-verbosity=1" >> ${BASEDIR}/my.cnf
-  fi
   echo "wsrep-provider=${BASEDIR}/lib/libgalera_smm.so" >> ${BASEDIR}/my.cnf
   echo "wsrep_sst_method=rsync" >> ${BASEDIR}/my.cnf
-  #echo "wsrep_sst_method=xtrabackup-v2" >> ${BASEDIR}/my.cnf
```

Comment:
In time we should any option viable here?

Reply:
yes we can use `mariabackup` for SST.